CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+Unreleased
+------
+- Replace css preprocessor, from sass to less
```

Comment:
Can we add links to `sass` and `less` repo pages?

Reply:
![Ditto](https://media.giphy.com/media/uQZTgSuGZMTHG/giphy.gif)